src/pages/index.tsx

```diff
@@ -117,7 +118,9 @@ export const getServerSideProps = withMultipleServerSideProps(
     }
 
     const resolvedByUsername =
-      validatedCaseState === "Resolved" && !currentUser.isSupervisor ? currentUser.username : undefined
+      validatedCaseState === "Resolved" && !currentUser.hasAccessTo[Feature.UnlockOtherUsersCases]
```

Comment:
I think it was more readable with the `!currentUser.isSupervisor` here since we use the `resolvedByUsername` var to filter resolved cases visible to the user

Reply:
Maybe we could define another feature something like `Feature.ListAllResolvedCases` to make it more explicit